lib/routes/account.js

```diff
@@ -213,7 +213,7 @@ module.exports = function (
               emailCode: account.emailCode,
               emailVerified: account.emailVerified,
               verifierSetAt: account.verifierSetAt,
-              createdAt: optionallyOverrideCreatedAt(),
+              createdAt: parseInt(query._createdAt),
```

Comment:
@philbooth I'm not sure about this change, before this would be blocked in prod, but is it allowing passing strings into the db query and request passes?

> curl 'http://127.0.0.1:9000/v1/account/create?_createdAt=wat' -H 'Referer: http://127.0.0.1:3030/' -H 'Origin: http://127.0.0.1:3030' -H 'User-Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_11_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/51.0.2704.103 Safari/537.36' -H 'Content-Type: application/json' --data-binary '{"email":"fxatester0.5167064475734202@restmail.net","authPW":"673fb6a8391158ba11dead916bb680222f9228778e2e21ce2b2120045b70b5af","resume":"eyJlbnRyeXBvaW50IjpudWxsLCJmbG93QmVnaW4iOjE0NzA0MDQ5NDYyOTAsImZsb3dJZCI6IjlkZWQ4MGE5ZjVlNDM1Y2ZkZTI3MDAxMDYwNjRhNGZlYmVkNjM3M2M1ZTUwMDhmZjllZTc2MDNlYzQwZWZjOGMiLCJyZXNldFBhc3N3b3JkQ29uZmlybSI6dHJ1ZSwidW5pcXVlVXNlcklkIjoiNDZhYjlmMGMtNTY0Yy00ODRmLWFlOGItZmFhYTEzYTEyNmI3IiwidXRtQ2FtcGFpZ24iOm51bGwsInV0bUNvbnRlbnQiOm51bGwsInV0bU1lZGl1bSI6bnVsbCwidXRtU291cmNlIjpudWxsLCJ1dG1UZXJtIjpudWxsfQ==","metricsContext":{"flowId":"9ded80a9f5e435cfde2700106064a4febed6373c5e5008ff9ee7603ec40efc8c","flowBeginTime":1470404946290}}' --compressed

****

> {"uid":"4b5ccf16c81c4559b5e421ff1dededa8","sessionToken":"e34146ad455d7f58fa9fb43dd0fec1d71471cc7743ef5c98c5ff9b462c8a0d97","authAt":1470405118}%                                                                                            



correct me if I'm wrong about this?

Also the promise resolves into a `NaN` and some proper `createdAt`, hmmm:

![image](https://cloud.githubusercontent.com/assets/128755/17438857/036b67e4-5af3-11e6-8940-4ffc15600eb6.png)